modules/core/src/main/java/org/apache/ignite/internal/binary/BinaryClassDescriptor.java

```diff
@@ -421,7 +421,7 @@ boolean isBinary() {
     /**
      * @return Described class.
      */
-    Class<?> describedClass() {
+    public Class<?> describedClass() {
```

Comment:
Can we use `BinaryUtils#resolveClass` instead of making this method public?

Reply:
Fixed, thank you.